numpy/add_newdocs.py

```diff
@@ -2926,10 +2926,14 @@ def luf(lamdaexpr, *args, **kwargs):
     """
     Tuple of array dimensions.
 
-    Notes
-    -----
-    May be used to "reshape" the array, as long as this would not
-    require a change in the total number of elements
+    The shape attribute is usually used to get at the current shape of an
+    array, but may also be used to reshape the array by assigning a different
+    tuple to it.  As with `numpy.reshape`, one of the new shape dimensions can
+    be -1, in which case its value is inferred from the length of the array and
+    remaining dimensions. This method of reshaping an array is not recommended,
```

Comment:
The sentence ordering here might lead the reader to think that this sentence refers simply to the `-1` behaviour.